editor/editor_node.cpp

```diff
@@ -2715,7 +2715,7 @@ void EditorNode::_menu_option_confirm(int p_option, bool p_confirmed) {
 					bool confirm = EDITOR_GET("interface/editor/quit_confirmation");
 					if (confirm) {
 						confirmation->get_ok()->set_text(p_option == FILE_QUIT ? TTR("Quit") : TTR("Yes"));
-						confirmation->set_text(p_option == FILE_QUIT ? TTR("Exit the editor?") : TTR("Open Project Manager?"));
+						confirmation->set_text(p_option == FILE_QUIT ? TTR("Exit the editor?") : TTR("Reload Current Project?"));
```

Comment:
That's not correct when using `RUN_PROJECT_MANAGER`. There are three possible options which should all have their wn text.